var/spack/repos/builtin.mock/packages/cmake-client-dep/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+def check(condition, msg):
+    """Raise an install error if condition is False."""
+    if not condition:
+        raise InstallError(msg)
+
+
+class CmakeClientDep(CMakePackage):
+    """A dummy package that uses cmake."""
```

Comment:
This docstring is generic. I recommend replacing it with a docstring (or `#`-style comment) that explains how it interacts with `cmake-client`.

Also, the `Dep` in `CmakeClientDep` I recommend expanding to `Dependent` (`dep` can expand to `dependent` or `dependency`; we occasionally use that abbreviation but AFAIK in places where it is an iteration variable and the context makes it clear like `for dep in dependencies`).

Reply:
Will do.